server/controllers/IndividualServices.js

```diff
@@ -54,15 +59,27 @@ module.exports.putLiveAirInterfaceTransimitterIsOn = function putLiveAirInterfac
 };
 
 module.exports.putLiveControlConstructExternalLabel = function putLiveControlConstructExternalLabel (req, res, next, body, mountName, user, originator, xCorrelator, traceIndicator, customerJourney) {
+  let startTime = process.hrtime();
+  let responseCode = responseCodeEnum.code.OK;
+  let responseBodyToDocument = {};
   IndividualServices.putLiveControlConstructExternalLabel(body, mountName, user, originator, xCorrelator, traceIndicator, customerJourney)
-    .then(function (response) {
-      utils.writeJson(res, response);
-    })
-    .catch(function (response) {
-      utils.writeJson(res, response);
-    });
+  .then(async function (responseBody) {
+    let mountName = req.url.match(/(?<=construct=)(\w)+/g);
+    let requrl =req.url.replace(mountName,"{mountName}" );
```

Comment:
In the "req" object , you can find the path defined in the property openapi.openApiRoute.
So , instead of translating the incoming url to the operation-name defined here, you shall do the following , 
```
let openApiPath = req.openapi.openApiRoute;
```